platform/linux-generic/include/odp_config_internal.h

```diff
@@ -125,7 +125,9 @@ extern "C" {
  * defined segment length (seg_len in odp_pool_param_t) will be rounded up into
  * this value.
  */
-#define CONFIG_PACKET_SEG_LEN_MIN CONFIG_PACKET_MAX_SEG_LEN
+#define CONFIG_PACKET_SEG_LEN_MIN ((2 * 1024) - \
+				   CONFIG_PACKET_HEADROOM - \
+				   CONFIG_PACKET_TAILROOM)
```

Comment:
Have we quantified the effect of this change on applications that use jumbo frames?

Reply:
Those applications should set pool parameters pkt.len, pkt.seg_len, etc. This config rounds up small len/seg_len parameters (e.g. 1.5k => 2k), but does not round down large parameters. E.g. 8k jumbo frame support remains the same if application has set pkt.len to 8k.